ubuntudesign/vanilla-framework
# Add examples

## Done

Added examples of each base, pattern and utility for local QA'ing and to become available for the docs when using the new documentation builder.

When checking the code. It may be easier to step through commit at a time to have piecemeal reviews.

## QA

- Pull down this branch
- Run `gulp develop`
- Then check a handful of examples in the browser

## Details
Fixes https://github.com/ubuntudesign/vanilla-framework/issues/779
